Lombiq.Hosting.Tenants.FeaturesGuard/Handlers/FeaturesEventHandler.cs

```diff
@@ -65,10 +65,9 @@ public async Task EnableConditionallyEnabledFeaturesAsync(IFeatureInfo featureIn
         }
 
         var allConditionFeatureIds = new List<string>();
-        foreach (var conditionFeatureId in conditionallyEnabledFeatures.Values)
```

Comment:
It's not a list, nor should you use such Hungarian notation if avoidable. In this case, it can simply be `conditionFeatureIds`.

Reply:
Don't think it's got anything to do with Hungarian notations, I would think it's the same as saying "shopping list"; it doesn't necessarily have to be "list of something". 

But it's not a list indeed so I'll change the name.